lib/fastlane/actions/xcake.rb

```diff
@@ -0,0 +1,27 @@
+module Fastlane
+  module Actions
+    class XcakeAction < Action
+      def self.run(params)
+        Actions.verify_gem!('xcake')
+        Actions.sh('xcake')
```

Comment:
While this works, you could potentially change this to

```ruby
require 'xcake'
Xcake::Command.run
```
or similar. 

What do you think? By using the Ruby code, the user will get the actual stack trace if something goes wrong.